src/freenas-installer/etc/install.sh

```diff
@@ -60,7 +60,16 @@ check_is_swap_safe()
     # If unset, we are either on TrueNAS or didn't find an unsafe disk.
     case "${SWAP_IS_SAFE:="YES"}" in
 	# Accept YES or NO (case-insensitive).
-	[Yy][Ee][Ss]) ;;
+	[Yy][Ee][Ss])
+	    # Confirm swap setup with FreeNAS users.
+	    if ! is_truenas &&
+		! dialog --clear --title "${AVATAR_PROJECT}" \
+		    --yes-label "Create swap" --no-label "No swap" --yesno  \
+		    "Create mirrored 16GB swap partitions on boot devices?" \
```

Comment:
Mention of "mirrored" here can be confusing for somebody who uses single boot device.  I'd remove that word.